README.md

```diff
@@ -67,7 +67,7 @@ Now, you have built the sample and run it, without writing any C++ code or openi
 - **We won't use [vcpkg](https://vcpkg.io/)** during this course.
 
 ## Awesome OpenGL Games:
-- [AntWar](https://github.com/YamanQD/AntWare/): Damascus university project 2021-2022
+- [AntWare](https://github.com/YamanQD/AntWare/): Damascus university project 2021-2022
```

Comment:
Isn't it a typo and you regret it?😅

Reply:
Oh no, it's like software, hardware... Antware

Reply:
Hmmmm, love it.. BTW, [this](https://www.reddit.com/r/opengl/comments/rev6gr/just_a_little_rant_about_my_college_still/) was very entertaining🥶.

Reply:
> Hmmmm, love it.. BTW, [this](https://www.reddit.com/r/opengl/comments/rev6gr/just_a_little_rant_about_my_college_still/) was very entertaining🥶.

Indeed 🥶